src/components/nav/Nav.js

```diff
@@ -43,6 +76,7 @@ const Nav = () => {
           </ul>
         </div>
       </div>
+      {signModal ? <LoginSignup handleLoginModal={handleLoginModal} /> : null}
```

Comment:
```suggestion
      {signModal && <LoginSignup handleLoginModal={handleLoginModal} />}
```
true일 경우만 처리한다면, 삼항연산자보다는 && 연산자가 더 간결하게 표현 가능합니다!

Reply:
리뷰 반영하여 수정했습니다!